packages/block-editor/src/components/block-tools/index.js

```diff
@@ -192,7 +192,7 @@ export default function BlockTools( {
 				{ /* Even if the toolbar is fixed, the block popover is still
 					needed for navigation and zoom-out mode. */ }
 				{ ! showEmptyBlockSideInserter && hasSelectedBlock && (
-					<SelectedBlockTools
+					<BlockToolbarPopover
 						__unstableContentRef={ __unstableContentRef }
 						clientId={ clientId }
```

Comment:
Is clientId useless here?

Reply:
It is used within the component, but we could fetch it from within the `<BlockToolbarPopover />` instead if you'd like. I had it passed it because it was already available, but if you prefer letting `<BlockToolbarPopover />` and `<EmptyBlockInserter />` handle them on their own, that's fine with me!